scripts/test_env.py

```diff
@@ -247,7 +247,7 @@ def setup(args: argparse.Namespace):
 
 def cleanup(out_dir: pathlib.Path):
     logging.info("Killing peer processes...")
-    subprocess.run(["pkill", "-9", "irohad"])
+    subprocess.run(["pkill", "-9", "iroha"])
```

Comment:
Is it right?

Reply:
it is because processes are named `iroha0` etc